src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/Serialization/TagHelperDescriptorJsonConverter.cs

```diff
@@ -4,14 +4,17 @@
 using System;
 using System.Collections.Generic;
 using Microsoft.AspNetCore.Razor.Language;
+using Microsoft.CodeAnalysis.Razor.Serialization.Internal;
 using Newtonsoft.Json;
 
 namespace Microsoft.CodeAnalysis.Razor.Serialization
 {
-    internal class TagHelperDescriptorJsonConverter : JsonConverter
+    internal partial class TagHelperDescriptorJsonConverter : JsonConverter
```

Comment:
Why partial?

Reply:
Oh, I think this happened when I extracted the class, will revert.